jenkins/scripts/integration_test.sh

```diff
@@ -122,6 +124,24 @@ fi
 TEST_EXECUTER_IP="$(openstack port show -f json "${TEST_EXECUTER_PORT_NAME}" \
   | jq -r '.fixed_ips[0].ip_address')"
 
+if [ "$OS_REGION_NAME" == "Fra1" ]
```

Comment:
Nit: It's more consistent to use `[[` everywhere instead of `[` if you don't need backwards compatibility with old shells. `[` is fine in this case, but it can be confusing because, for example, pattern matching only works with `[[` not with `[`.

Reply:
That was failing so I had to change it to single `[`